rules/linux/file_create/file_create_lnx_triple_cross_rootkit_lock_file.yml

```diff
@@ -0,0 +1,21 @@
+title: Triple Cross eBPF Rootkit Default LockFile
+id: c0239255-822c-4630-b7f1-35362bcb8f44
+status: experimental
+description: Detects the creation of the file "rootlog" which is used by the TripleCross rootkit as a way to check if the backdoor is already running.
+references:
+    - https://github.com/h3xduck/TripleCross/blob/master/src/helpers/execve_hijack.c#L33
```

Comment:
https://github.com/h3xduck/TripleCross/blob/1f1c3e0958af8ad9f6ebe10ab442e75de33e91de/src/helpers/execve_hijack.c#L33

Nore sure use If the soc operator don't read C First one should be enough
https://github.com/h3xduck/TripleCross/blob/1f1c3e0958af8ad9f6ebe10ab442e75de33e91de/src/helpers/execve_hijack.c#L305

Reply:
Okay. I'll remove the second reference.

Reply:
Same here as we put the line , I have get the permanent link